GymApp/gym-app/gym-app/src/pages/PassesPage.jsx

```diff
@@ -31,28 +22,16 @@ const PassesPage = ({ storedAuthToken }) => {
 
   const handleAddFormSubmit = (e) => {
     e.preventDefault()
-    const { date, temperatureC, temperatureF, summary } = addFormData
-    const pass = {
-      id: uuid(),
-      date: date,
-      temperatureC: temperatureC,
-      temperatureF: temperatureF,
-      summary: summary,
-    }
+    const pass = { id: uuid(), ownerId: uuid(), ...addFormData }
```

Comment:
I dont like using random Uuids here, I think they should be added by backend and passed as null here. Usually you use DTOs here, which do not have neither ID nor owner ID. What kind of values does swagger support here?